lib/ui/tab-navigation.tsx

```diff
@@ -0,0 +1,137 @@
+import { cn } from "@/lib/utils"
+import * as NavigationMenuPrimitives from "@radix-ui/react-navigation-menu"
+import { cva, type VariantProps } from "class-variance-authority"
+import { motion } from "framer-motion"
+import * as React from "react"
+
+function getSubtree(
+  options: { asChild: boolean | undefined; children: React.ReactNode },
+  content: React.ReactNode | ((children: React.ReactNode) => React.ReactNode)
+) {
+  const { asChild, children } = options
+  if (!asChild)
+    return typeof content === "function" ? content(children) : content
+
+  const firstChild = React.Children.only(children) as React.ReactElement
+  return React.cloneElement(firstChild, {
+    children:
+      typeof content === "function"
+        ? content(firstChild.props.children)
+        : content,
+  })
+}
+
+const tabNavigationVariants = cva(
+  "flex items-center justify-start gap-1 overflow-x-auto whitespace-nowrap border-b border-b-f1-border-secondary px-6 py-3 [scrollbar-width:none] [&::-webkit-scrollbar]:hidden",
```

Comment:
@nlopin not sure what's going on here, if I set the border this way, it is not displayed (that's why I was doing this weird thing with border-transparent etc earlier) 😅

Reply:
are you sure `border-b` does something?

Reply:
It does, but i was missing the solid type for the border. There's no way to add it only to the border bottom with Tailwind without adding an extended class, so I fixed it [like we do in the Breadcrumbs component](https://github.com/factorialco/factorial-one/blob/main/lib/components/Navigation/Breadcrumb/index.tsx), setting every other border to 0.